core/base-service/check-error-response.js

```diff
@@ -25,6 +26,13 @@ export default function checkErrorResponse(httpErrors = {}) {
         error = new InvalidResponse(props)
       }
     }
+
+    if (res.statusCode === 429) {
+      log.error(
+        new Error(`429 Too Many Requests calling ${res.requestUrl.origin}`),
```

Comment:
Using `res.requestUrl.origin` instead of `res.requestUrl.toString()` here will only log the domain part of the request. This is useful for two reasons:

1. It will mean Sentry considers every "429 Too Many Requests calling https://example.com" to be the same issue for de-duplication purposes - we'll log them all as many events attached to one issue. If we used the whole URL then "429 Too Many Requests calling https://example.com/foo" and "429 Too Many Requests calling https://example.com/bar" would be considered different issues, leading to a lot of noise.
2. We won't send any URL parameters (which could include an API key or token for services that use query string auth) to Sentry

Just the domain should be enough information to tell us which upstream service is applying a rate limit though.